sql/core/src/main/scala/org/apache/spark/sql/DataFrameImpl.scala

```diff
@@ -179,10 +179,20 @@ private[sql] class DataFrameImpl protected[sql](
     select((col +: cols).map(Column(_)) :_*)
   }
 
+  override def selectExpr(exprs: String*): DataFrame = {
```

Comment:
yea - but asking users to wrap a column name in backticks in strings is fairly annoying.